--- sblim-smis-hba-1.0.0/src/Linux_CommonHBA.h.orig	2009-09-01 15:22:39.000000000 +0200
+++ sblim-smis-hba-1.0.0/src/Linux_CommonHBA.h	2015-01-12 21:38:40.318262378 +0100
@@ -33,7 +33,7 @@
 /* ---------------------------------------------------------------------------*/
 
 #include <time.h>
-#include <sys/hbaapi.h>
+#include <hbaapi.h>
 
 #define HBA_MAX_DEV_NAME        30
 #define DEVICEID_SIZE           256
